mobile/lib/pages/editing/crop.page.dart

```diff
@@ -0,0 +1,185 @@
+import 'package:flutter/material.dart';
+import 'package:crop_image/crop_image.dart';
+import 'edit.page.dart';
+
+class CropImagePage extends StatefulWidget {
+  final Image image;
+  const CropImagePage({super.key, required this.image});
+  @override
+  State<CropImagePage> createState() => _CropImagePageState();
+
+}
+class _CropImagePageState extends State<CropImagePage> {
+  late CropController _cropController;
+  double? _selectedAspectRatio;
+  @override
+  void initState() {
+    super.initState();
+    _cropController = CropController(
+      defaultCrop: const Rect.fromLTRB(0.1, 0.1, 0.9, 0.9),
+    );
+  }
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        backgroundColor: Colors.black,
+        leading: const CloseButton(color: Colors.white),
+        actions: [
+          IconButton(
+            icon: const Icon(Icons.done_rounded, color: Colors.white, size: 24),
+            onPressed: () async {
+              Image croppedImage = await _cropController.croppedImage();;
+              Navigator.push(
+                context,
+                MaterialPageRoute(
+                  builder: (context) => EditImagePage(imageSource: croppedImage),
+                ),
+              );
+            },
+          ),
+        ],
+      ),
+      body: Column(
+        children: [
+          Container(
+            padding: const EdgeInsets.only(top: 20),
+            width: MediaQuery.of(context).size.width * 0.8,
+            height: MediaQuery.of(context).size.height * 0.6,
+            child: CropImage(
+              controller: _cropController,
+              image: widget.image,
+              gridColor: Colors.white,
+            ),
+          ),
+          Expanded(
+            child: Container(
+              width: double.infinity,
```

Comment:
Shouldn't be necessary with `Expanded`.

Reply:
No, I need the container to stop the change in layout of my bottom tools bar when rotated the image. So I cant really remove the container in there.